plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/FileSystemExchangeSink.java

```diff
@@ -84,14 +85,16 @@ public FileSystemExchangeSink(
             Optional<SecretKey> secretKey,
             int maxPageStorageSize,
             int exchangeSinkBufferPoolMinSize,
-            int exchangeSinkBuffersPerPartition)
+            int exchangeSinkBuffersPerPartition,
+            long exchangeSinkFileTargetSize)
```

Comment:
nit: `exchangeSink` prefix is not necessary here. Also in other places we try to add a prefix that mentions the unit, for example `exchangeSinkFileTargetSizeInBytes`